app/src/main/java/com/sunasterisk/getitdone/ui/home/HomeFragment.kt

```diff
@@ -0,0 +1,65 @@
+package com.sunasterisk.getitdone.ui.home
+
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuInflater
+import androidx.appcompat.app.AppCompatActivity
+import com.sunasterisk.getitdone.R
+import com.sunasterisk.getitdone.base.BaseFragment
+import com.sunasterisk.getitdone.data.model.Task
+import com.sunasterisk.getitdone.utils.toast
+import kotlinx.android.synthetic.main.fragment_home.*
+
+class HomeFragment : BaseFragment<HomeContract.View, HomePresenter>(), HomeContract.View {
+
+    override val layoutId get() = R.layout.fragment_home
+    override val presenter get() = HomePresenter.getInstance()
+
+    private var listId = 0
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        arguments?.let { listId = it.getInt(ARG_LIST_ID) }
+        (activity as AppCompatActivity).setSupportActionBar(home_toolbar)
+        setHasOptionsMenu(true)
+    }
+
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
+        inflater.inflate(R.menu.home_fragment_search_menu, menu)
+        super.onCreateOptionsMenu(menu, inflater)
+    }
+
+    override fun initComponents(savedInstanceState: Bundle?) {
+        presenter.getTaskListFromId(listId)
+        presenter.getTasksFromListId(listId)
```

Comment:
2 cái command này hại não nhỉ  :>>

Reply:
để em sửa lại thế này ạ, đọc vẫn hơi hại não 1 tí :))
```suggestion
        presenter.getTaskListFromId(id)
        presenter.getTasksFromTaskListId(listId)
```